test/e2e/README.md

```diff
@@ -0,0 +1,55 @@
+# Running End-to-End Tests
+
+To run end-to-end test cases on your system, follow these steps:
```

Comment:
why is `dapp-inter` documenting this?  shouldn't all this be in `@agoric/synpress` ?

Reply:
It's more so of a quick guide to setting it up and running tests on `dapp-inter`. 
`@agoric/synpress` has more detailed documentation.

Reply:
> It's more so of a quick guide to setting it up and running tests on dapp-inter.

Then they shouldn't have to **Install Dependencies**, right?

And why should they have to paste some Docker command? I'd expect any commands they need to run for tests to begin with `yarn`.